### PR TITLE
Update meeting-policies-participants-and-guests.md

### PR DESCRIPTION
**Pull request by dipinn26592**
Known limitation with DL.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-participants-and-guests.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ This is a per-organizer policy. This setting controls whether people join a meet
...
 |**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "Only me".          |
-|**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "People I invite".          |
+|**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "People I invite". User added as a part of Distribution List will have to go through Lobby.      |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-participants-and-guests.md:66]:
```suggestion
|**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "People I invite". Users added as a part of a distribution group will have to go through the lobby.      |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dipinn26592, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**